packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -117,6 +117,18 @@ - (instancetype)initWithFrame:(CGRect)frame
     if ([initialUrl isKindOfClass:[NSString class]]) {
       [self loadUrl:initialUrl];
     }
+    
+    
+    BOOL hasPullToRefresh = [args[@"hasPullToRefresh"] boolValue];
+
+    if (hasPullToRefresh) {
+      UIRefreshControl *refreshControl = [[UIRefreshControl alloc] init];
+      [refreshControl setTintColor:[UIColor grayColor]];
```

Comment:
I don't think we should assume a default tint color for user. We can make the color configurable with an argument from dart.